tests/codeception/acceptance/StanfordEventsSeriesCest.php

```diff
@@ -66,8 +66,8 @@ protected function createEventSeriesNode(AcceptanceTester $I, $node_title = NULL
       $event_nodes[] = ['target_id' => $node->id()];
     }
 
-    // Something sticks.
-    $I->runDrush("cr");
+    drupal_static_reset();
+    drupal_flush_all_caches();
```

Comment:
@pookmish When installing a module during the test run it appears that the Drupal internal registries fail to update with the information. This is a HARD reset but it gets all the ducks in a row and allows the entity to be created below.

Reply:
Should we move that to caravan execution then?

Reply:
I don't think it is needed most times. This is really only for circleci when we enable modules with entities during the test run.

Reply:
sounds good enough to me.
that `@depends` is new to me. but what can be used instead is the `_before` method. see https://codeception.com/docs/07-AdvancedUsage

Reply:
Oh, I like that better. I will change that out.